portfolio/src/main/webapp/script.js

```diff
@@ -61,3 +61,28 @@ const liElement = document.createElement('li');
 liElement.innerText = text;
 return liElement;
 }
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Creates a chart and adds it to the page. */
+function drawChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Animal');
+  data.addColumn('number', 'Count');
+        data.addRows([
```

Comment:
[4.2](https://google.github.io/styleguide/jsguide.html#formatting-block-indentation): Each time a new block or block-like construct is opened, the indent increases by two spaces. When the block ends, the indent returns to the previous indent level. The indent level applies to both code and comments throughout the block. 

The indentation in not uniform here